lib/matplotlib/figure.py

```diff
@@ -462,7 +459,12 @@ def show(self, warn=True):
     def _get_axes(self):
         return self._axstack.as_list()
 
-    axes = property(fget=_get_axes, doc="Read-only: list of axes in Figure")
+    axes = property(fget=_get_axes,
+                    doc="List of axes in the Figure. You can access and "
+                        "modify the axes in the Figure through this list. "
+                        "Do not modify the list itself. Instead, use "
```

Comment:
the second and third sentence are contradicting each other.

Reply:
No. Assume `axs = fig.axes` being `[ax1, ax2]`. You can access and modify `ax1`, e.g. axs[0].set_xlim(0, 1)`. But you cannot add or remove an axes from the figure by `fig.axes.append(ax3)` or similar.

Suggestions how to describe this more clearly are welcome.

Reply:
just remove the "and modify" in the first sentence?  it's not as if python had a notion of const reference anyways...